examples/saving_and_loading_data/coordinates_in_asdf.py

```diff
@@ -68,7 +68,7 @@ def r_2_func(x):
 
     return SkyCoord(
         x=r.to(u.cm) * np.sin(phi.to(u.radian)),
-        y=u.Quantity(r.shape[0] * [0 * u.cm]),
+        y=r.shape[0] * u.cm * 0,
```

Comment:
What is this doing!?

Reply:
Even simpler:
```python
np.zeros_like(r)
```

Reply:
(The code as written works because the `0 * u.cm` gets broadcasted anyway, so you could just replace it all with a scalar Quantity.)